modules/virt-querying-metrics.adoc

```diff
@@ -31,6 +31,14 @@ A value above '0' means that the vCPU wants to run, but the host scheduler canno
 ====
 To query the vCPU metric, the `schedstats=enable` kernel argument must first be applied to the `MachineConfig` object. This kernel argument enables scheduler statistics used for debugging and performance tuning and adds a minor additional load to the scheduler.
 ====
+The `kubevirt_vmi_vcpu_delay_seconds_total` metric reports the cumulative time that a vCPU was enqueued by the host scheduler but could not run immediately. This delay is exposed to the virtual machine (VM) as steal time and can affect performance, often indicating CPU overcommitment or contention on the host.
```

Comment:
"This delay is exposed to the virtual machine (VM)"

You're absolutely correct to add (VM) after the first use of virtual machine. In this case, it should actually be added in the earlier existing section. For some reason, the first use of virtual machine  in this module did not have (VM) added. I would remove it from your addition if you can, and edit line 14 in the note.